unison.opam

```diff
@@ -11,10 +11,9 @@ bug-reports: "https://github.com/bcpierce00/unison/issues"
 dev-repo: "git://github.com/bcpierce00/unison.git"
 build: ["dune" "build" "-p" name "-j" jobs]
 depends: [
-  "ocaml" {build & >= "4.08"}
-  "ocamlfind" {build}
-  "dune" {build & >= "2.3"}
-  "lablgtk3" {build & >= "3.1.0"}
+  "ocaml" {>= "4.08"}
```

Comment:
Please bear with me as i am not clueful about opam but it's becoming clear somebody in unison needs to be.  As I read the docs, this is declaring that ocaml (which has compiler and runtime) is necessary at build time, but not at run time.   And the commit drops `build &` to indicate that it is necessary at run time.   In my build (with pkgsrc, not opam, NetBSD amd64), the unison binary does not depend on any compiler files.  But, it's a native build, not byte code, and it seems possible that a byte code build could be done, in which case I would expect a runtime lib would be needed.

Therefore, *in general*, ocaml is needed at run time.

Is that right?